test/Libraries/PackageManagerTests/PackageUploadBuilderTests.cs

```diff
@@ -29,6 +32,68 @@ private IPackageUploadBuilder BigPackageUploadBuilderMock()
             return new PackageUploadBuilder(pdb.Object, zipper.Object);
         }
 
+        private class directoryTestClass : IDirectoryInfo
+        {
+            public string FullName { get; set; }
+
+        }
+
+        #endregion
+
+        #region GregFileUtilsTests
+            [Test]
+            public void FileCompressorZipsSamplePackageDirectoryToValidArchive()
+        {
+            var executingLocation = new DirectoryInfo(Assembly.GetExecutingAssembly().Location);
+            var testDirectory = Path.Combine(executingLocation.Parent.Parent.Parent.Parent.FullName, "test");
```

Comment:
where is it pointing?

Reply:
talked about this with @QilongTang yesterday, this is pointing back up the /test/pkgs folder.